connect_ext_ppr/models/configuration.py

```diff
@@ -12,20 +12,22 @@ class Configuration(Model):
 
     PREFIX = 'CFL'
 
+    STATE = ConfigurationStateChoices
```

Comment:
There is a lot of refactoring in several files. I think it is more convenient to make a separate commit for refactoring.